src/main/java/org/folio/consortia/domain/entity/SharingRoleEntity.java

```diff
@@ -3,21 +3,29 @@
 import jakarta.persistence.Entity;
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
 import org.folio.consortia.domain.entity.base.AuditableEntity;
 
 import java.util.UUID;
 
 @Data
```

Comment:
`@RequiredArgsConstructor` can be removed here, `@Data` implicitly contains it (https://projectlombok.org/features/Data).

Reply:
Thanks, removed

Reply:
saw, error and to be on safe side reverted back requiredArgcontructor